config/locales/views/en.yml

```diff
@@ -325,46 +325,59 @@ en:
+        intro: 'Potential reasons to submit a report to the Policy & Abuse team:'
+        violation: 'You have encountered content on the Archive that violates the %{tos_link}.'
+        content: 'You would like to know whether specific content is allowed on the Archive.'
+        harassment: 'You have experienced or witnessed harassment on the Archive.'
+        hack: 'You believe that your account was hacked.'
+        suspended: Y'ou have been suspended or your work has been removed, and you have not received an email explaining why.'
+      include:
+        intro: 'What to include in your report description:'
+        username: 'the username of the person you are reporting'
+        links: 'links to any of their other content that you would like to report'
+        quote: 'a quote from or summary of the reported content explaining why it violates the %{tos_link}'
+        sources: 'links to any relevant sources or screenshots'
+        languages: 'We can answer Abuse reports in %{list}.'
+        delay: 'There will be an additional delay for responses in any language other than English.'
+        ao3: 'AO3 Terms of Service'
+      do_not_spam: 'Please only report one user at a time, and do not submit multiple reports about the same user. Our volunteer team is small, so it may take several months for us to get to your report.'
       form:
         legend:
-          abuse: Link and Describe Abuse
+          abuse: 'Link and Describe Abuse'
         landmark:
-          send: Send to Abuse Team
+          send: 'Send to Abuse Team'
         submit:
-          active: Submit
+          active: 'Submit'
         name:
-          label: Your name (optional)
+          label: 'Your name or username (optional)'
         email:
-          label: Your email (required)
-          description: We cannot act on reports without a valid email address.
+          label: 'Your email (required)'
+          description: 'We cannot contact you if the email address you provide is invalid.'
         language:
-          label: Select language (required)
+          label: 'Select language (required)'
         summary:
-          label: Brief summary of Terms of Service Violation (required)
-          error: Please enter a brief summary of your message
+          label: 'Brief summary of Terms of Service violation (required)'
+          error: 'Please enter a brief summary of your message'
+          description: 'For example, "harassment", "not a fanwork", "commerical activities", etc'
         comment:
-          label: Your comment (required)
-          error: Please describe your concern.
-          description: Please include all relevant URLs and what about the content violates the Archive %{tos_link}. The more thorough the information the quicker we can start investigating.
-          tos: Terms of Service
+          label: 'Description of the content you are reporting (required)'
```

Comment:
It looks like you'll need to update this in the cucumber test as well: https://github.com/otwcode/otwarchive/blob/master/features/other_a/abuse_report.feature#L13